test/Forwarder.js

```diff
@@ -267,9 +267,11 @@ describe("Forwarder tests", () => {
     });
 
     it("Client shouldn't be allowed to getResult if stake is less than minStake", async () => {
```

Comment:
Add a test case for calling to get result when above minStake and how changing the minStake param changes whitelist logic

Reply:
Resolved in [223b3ba](https://github.com/razor-network/bridge-proxy/pull/50/commits/223b3bae899e0fac17a245c8e8c103799456f9dd)